<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберите экран, на который будут транслироваться результаты:</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="screen1">
     <property name="text">
      <string>Экран 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="screen2">
     <property name="text">
      <string>Экран 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="screen3">
     <property name="text">
      <string>Экран 3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="screen4">
     <property name="text">
      <string>Экран 4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="screen5">
     <property name="text">
      <string>Экран 5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="screen6">
     <property name="text">
      <string>Экран 6</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
